web/gds-user-ui/src/modules/dashboard/member/components/MemberModal/MemberModal.tsx

```diff
@@ -19,28 +19,36 @@ import { useFetchMember } from '../../hooks/useFetchMember';
 import { Trans } from '@lingui/macro';
 import { useSelector } from 'react-redux';
 import { memberSelector } from '../../member.slice';
+// import useToast from 'hooks/useToast';
 interface MemberModalProps {
   isOpen: boolean;
   onClose: () => void;
   member: any;
 }
 const MemberModal = ({ isOpen, onClose, member: memberId }: MemberModalProps) => {
   const network = useSelector(memberSelector).members.network;
-  const { member, isFetchingMember } = useFetchMember({ vaspId: memberId, network });
+  const { member, isFetchingMember, error, wasMemberFetched } = useFetchMember({
+    vaspId: memberId,
+    network
+  });
+
+  if (error) {
+    console.log('[] error', error?.response?.data);
+    // close the modal for now , later we need to show a toast message
```

Comment:
Would it be possible to return a toast error with the release?

Reply:
A story have been already created to Add it in a different PR.